sdk/ai/azure-ai-projects/tests/agents/test_agents_client.py

```diff
@@ -216,6 +272,103 @@ def test_update_agent(self, **kwargs):
         print("Deleted agent")
         client.close()
 
+    # test update agent without body: JSON
+    @agentClientPreparer()
+    @recorded_by_proxy
+    def test_update_agent_2(self, **kwargs):
+        # create client
+        client = self.create_client(**kwargs)
+        assert isinstance(client, AIProjectClient)
+
+        # create body for agent
+        body = {
+            "name": "my-agent",
+            "model": "gpt-4o",
+            "instructions": "You are helpful agent"
+        }
+
+        # create agent
+        agent = client.agents.create_agent(body=body)
+        assert agent.id
+        print("Created agent, agent ID", agent.id)
+
+        # update agent and confirm changes went through
+        agent = client.agents.update_agent(agent.id, name="my-agent2")
+        assert agent.name
+        assert agent.name == "my-agent2"
+
+        # delete agent and close client
+        client.agents.delete_agent(agent.id)
+        print("Deleted agent")
+        client.close()
+
+    # NOTE update_agent with overloads isn't working
```

Comment:
please create issues on tests that are not working to backlog

Reply:
sorry, I meant to delete that note